pkg/controller/migcluster/remote_watch.go

```diff
@@ -78,8 +57,8 @@ func StartRemoteWatch(r *ReconcileMigCluster, config RemoteManagerConfig) error
 	// Add remoteWatcher to remote MGR
 	log.Info("[rWatch] Adding controller to manager")
 	forwardEvent := event.GenericEvent{
-		Meta:   config.ParentResource.GetObjectMeta(),
-		Object: config.ParentResource,
+		Meta:   config.ParentMeta,
```

Comment:
Also had to switch from passing MigCluster as the parent resource to just passing ParentMeta and ParentObject to fix a dep cycle.